src/core/components/layout/components/container/container.tsx

```diff
@@ -1,27 +1,49 @@
 import { HTMLAttributes, ReactNode } from 'react';
 import { SerializedStyles } from '@emotion/react';
-import { container, containerBorder } from './styles';
+import {
+	container,
+	containerSideBorders,
+	containerTopBorder,
+	containerBackground,
+} from './styles';
 import { Props } from '@guardian/src-helpers';
+import { border } from '@guardian/src-foundations/palette';
 
 interface Container extends HTMLAttributes<HTMLDivElement>, Props {
-	border?: boolean;
+	sideBorders?: boolean;
+	topBorder?: boolean;
+	backgroundColor?: string;
+	borderColor?: string;
 	cssOverrides?: SerializedStyles | SerializedStyles[];
 	children: ReactNode;
 }
 
 const Container = ({
-	border = false,
+	sideBorders = false,
+	topBorder = false,
+	backgroundColor,
+	borderColor = border.secondary,
 	cssOverrides,
 	children,
 	...props
 }: Container) => {
 	return (
-		<div
-			css={[container, border ? containerBorder : '', cssOverrides]}
-			{...props}
+		<section
+			css={[backgroundColor && containerBackground(backgroundColor)]}
 		>
-			{children}
-		</div>
+			<div
+				css={[
+					container,
+					backgroundColor && containerBackground(backgroundColor),
+					topBorder && containerTopBorder(borderColor),
+					sideBorders && containerSideBorders(borderColor),
+					cssOverrides,
+				]}
+				{...props}
```

Comment:
It has been the convention in other components to spread props over the parent element of a component. Is there a reason we shouldn't spread the props over the `section` element?

Reply:
Happy to follow convention